src/rill.h

```diff
@@ -117,27 +120,33 @@ const char * rill_store_file(const struct rill_store *store);
 unsigned rill_store_version(const struct rill_store *store);
 rill_ts_t rill_store_ts(const struct rill_store *store);
 size_t rill_store_quant(const struct rill_store *store);
-size_t rill_store_keys(const struct rill_store *store);
-size_t rill_store_vals(const struct rill_store *store);
+size_t rill_store_keys_count(const struct rill_store *store, enum rill_column_e column);
 size_t rill_store_pairs(const struct rill_store *store);
 
+size_t __rill_store_data_off(const struct rill_store *store, enum rill_column_e column);
+size_t __rill_store_index_off(const struct rill_store *store, enum rill_column_e column);
```

Comment:
I would prefer if internal functions were not made visible in the main interface. That being said, I have a feeling that this is there mostly for debugging and will eventually be removed in which case it's fine.

Reply:
It is indeed a debugging function, and I can remove it later. If you see in rill_dump, I call these by passing a debug flag. (these were some minor sanity checks for me).

Reply:
Note that I tend to add a `dbg_` prefix to debug function to make it clear that they're for debugging and should eventually be removed.